kolibri/core/assets/src/utils/navItems.js

```diff
@@ -0,0 +1,11 @@
+export let menuItems = [];
+
+export function setMenuItems(items) {
+  menuItems = items;
+}
+
+export let userMenuItems = [];
```

Comment:
Well, doing it like this allows access to each item directly from the module namespace, whereas exporting an object as the default export from a module (which would allow the same thing) breaks our linting rules.